files/initramfs-tools/union-mount.j2

```diff
@@ -109,7 +109,10 @@ mount --bind ${rootmnt}/host/$image_dir/boot ${rootmnt}/boot
 ## Mount loop device or tmpfs for /var/log
 onie_platform=""
 aboot_platform=""
-. ${rootmnt}/host/machine.conf
+if [ -f ${rootmnt}/host/machine.conf ]; then
```

Comment:
this is not perfect. if somehow host/machine.conf is missing, then we will have issue.

Reply:
If any onie_* parameters are accessed, the host_machine.conf extraction will have to be moved here.. but only vendor specific parameters are accessed.